packages/next-auth/src/providers/mercado-libre.js

```diff
@@ -0,0 +1,48 @@
+/** @type {import(".").OAuthProvider} */
```

Comment:
New providers should to go under https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers

Could you please convert this to typescript as well, and add comments? Example: https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/github.ts